Automattic/wp-calypso
# Fix/sharing button states

## Changes proposed in this Pull Request

In https://github.com/Automattic/wp-calypso/pull/31123#issuecomment-469535497 @cburton4 advised that only one button per screen should have the primary (pink) state. 

I first attempted to make this change in https://github.com/Automattic/wp-calypso/pull/31123, but @simison wisely suggested that this change could ship separately.

### Before

![](https://d1jfzjx68gj8xs.cloudfront.net/items/1z3l3R2W1H1U423t3W11/Screen%20Shot%202019-03-07%20at%205.08.12%20AM.png)

### After

![](https://d1jfzjx68gj8xs.cloudfront.net/items/151o3L1f3u0A0Y280S0b/Screen%20Shot%202019-03-07%20at%205.07.13%20AM.png)

## Testing instructions

- Check out branch
- Compile changes
- View http://calypso.localhost:3000/sharing/lessbloat.wordpress.com
